src/python/turicreate/test/test_one_shot_object_detector.py

```diff
@@ -247,6 +247,18 @@ def test_summary(self):
         model = self.model
         model.summary()
 
+    def test_summary_str(self):
+        model = self.model
+        self.assertTrue(isinstance(model.summary("str"), str))
+
+    def test_summary_dict(self):
+        model = self.model
+        self.assertTrue(isinstance(model.summary("dict"), dict))
+
+    def test_summary_invalid(self):
+        model = self.model
+        model.summary("invalid")
```

Comment:
It seems like this should throw an exception.

Reply:
For all the toolkits now for invalid inputs,
they return print(self.__repr__()).
Do I need to change the logic for all toolkits?

Reply:
Let's not address that in this PR. Please file a GitHub issue for this problem with all toolkits.

Reply:
good!